src/main/java/org/opensearch/ad/transport/ValidateAnomalyDetectorTransportAction.java

```diff
@@ -186,17 +191,24 @@ protected DetectorValidationIssue parseADValidationException(ADValidationExcepti
             case NAME:
             case CATEGORY:
             case DETECTION_INTERVAL:
+                if (exception.getAspect().equals(ValidationAspect.MODEL)) {
+                    intervalSuggestion = exception.getIntervalSuggestion();
+                }
             case FILTER_QUERY:
             case TIMEFIELD_FIELD:
             case SHINGLE_SIZE_FIELD:
             case WINDOW_DELAY:
+                if (exception.getAspect().equals(ValidationAspect.MODEL)) {
+                    intervalSuggestion = exception.getIntervalSuggestion();
+                }
```

Comment:
Seems the added logic could be added before or after the switch statements, and doesn't have to do with the exception type. Also, this is duplicate logic with no breaks, so in the case of the type being `DETECTION_INTERVAL` this would get executed twice